ICRPG extended/ICRPGextended.html

```diff
@@ -0,0 +1,594 @@
+<!-- Compiled and minified CSS -->
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.98.2/css/materialize.min.css">
+<link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet">
+
+
+<div class="container">
+<div class="background">
+
+<br />
+<div id="main">
+    <div class='sheet-3colrow'>
+        <div class='sheet-col'>
+          <br/>
+          <br/>
+          <br/>
+          <label>Name:</label><input type="text" name="attr_name"><br />
+          <br />
+          <label>Class:</label><input type="text" name="attr_class">
+        </div>
+        <div class='sheet-col'>
+        <br/>
+        <br/>
+        <br/>
+        <label>Bioform:</label><input type="text" name="attr_bioform"><br />
+        <br /> 
+        <label>Mage Class and Kind:</label><input type="text" name="attr_mage">
+        </div>
+        <div class='sheet-col'>
+            <tr>
+                <input type="image" alt="for use with ICRPG"  style="margin-top:-30px; border-color: white;" src="https://cdn.discordapp.com/attachments/661523687643676673/676095795455328257/icrpg_logo.png" >
+            </tr>
+        </div>
+    </div>
+    <div class='sheet-col'>
+        <label>Origin:</label><input type="text" name="attr_origin" style="width:680px"><br />
+        <br />
+        <label>Destiny:</label><input type="text" name="attr_destiny" style="width:680px"><br />
+        <br />
+        <label>Story:</label><textarea name="attr_story" style="width:670px"></textarea><br />
+</div>
+<br />
+<br />
+
+
+
+<div class='sheet-3colrow'>
+  <div class='sheet-col'>
+        <!-- Put the content for the first column here -->
+    <div id="attributes">
+      <h3 style="text-align: center; color: #FFF; background-color: #c00607;">Attributes</h3><br />
+                <table style="width: 50%;">
```

Comment:
We no longer allow `<table>` to be used for basic layout per the Building Character Sheet guidelines.